[nlopt] build error on indigo

```
[nlopt] ./nlopt_optimize-oct.cc:32:31: error: Octave_map was not declared in this scope                                        
[nlopt]  static int struct_val_default(Octave_map &m, const std::string& k,                                                    
[nlopt]                                ^                                                                                       
[nlopt] ./nlopt_optimize-oct.cc:32:43: error: m was not declared in this scope                                                 
[nlopt]  static int struct_val_default(Octave_map &m, const std::string& k,                                                    
[nlopt]                                            ^                                                                           
[nlopt] ./nlopt_optimize-oct.cc:32:46: error: expected primary-expression before const                                         
[nlopt]  static int struct_val_default(Octave_map &m, const std::string& k,                                                    
[nlopt]                                               ^                                                                        
[nlopt] ./nlopt_optimize-oct.cc:33:6: error: expected primary-expression before int                                            
[nlopt]       int dflt)                                                                                                        
[nlopt]       ^                                                                                                                
[nlopt] ./nlopt_optimize-oct.cc:33:14: error: expression list treated as compound expression in initializer [-fpermissive]     
[nlopt]       int dflt)                                                                                                        
[nlopt]               ^                                                                                                        
[nlopt] ./nlopt_optimize-oct.cc:34:1: error: expected , or ; before { token                                                    
[nlopt]  {                                                                                                                     
[nlopt]  ^                                                                                                                     
[nlopt] make[7]: *** [nlopt_optimize.oct] Error 1                                                                              
[nlopt] make[6]: *** [all] Error 2                                                                                             
[nlopt] make[5]: *** [all-recursive] Error 1                                                                                   
[nlopt] make[4]: *** [all] Error 2                                                                                             
[nlopt] make[3]: *** [installed] Error 2                                                                                       
[nlopt] make[2]: *** [/home/wkentaro/ros/indigo/devel/lib/libnlopt_cxx.so] Error 2                                             
[nlopt] make[1]: *** [CMakeFiles/libnlopt_cxx.dir/all] Error 2                                                                 
[nlopt] make: *** [all] Error 2                                                                                                
[nlopt] <== '/home/wkentaro/ros/indigo/build/nlopt/build_env.sh /usr/bin/make --jobserver-fds=3,5 -j' failed with return code '2' 
Failed   <== nlopt [ 3.9 seconds ]                                                                                             
[build] There were '1' errors:                                                                                                 

Failed to build package 'nlopt' because the following command:

```

full build log: https://gist.github.com/wkentaro/bab6aaa8124462f9804a

--- a/3rdparty/nlopt/Makefile
+++ b/3rdparty/nlopt/Makefile
@@ -10,7 +10,7 @@ MK_DIR = $(shell rospack find mk)
 include $(MK_DIR)/download_unpack_build.mk
 
 installed: $(SOURCE_DIR)/unpacked
-	(cd $(SOURCE_DIR) && ./configure --enable-shared --with-cxx --prefix=$(DSTDIR) LIBS='-lstdc++' && make && make install)
+	(cd $(SOURCE_DIR) && ./configure --enable-shared --with-cxx --prefix=$(DSTDIR) LIBS='-lstdc++' --without-octave && make && make install)
 	touch installed
 clean:
 	-rm -rf include lib share $(SOURCE_DIR) installed

